tests/rds/test_rds_actions.py

```diff
@@ -237,3 +243,12 @@ def test_delete_db_cluster_endpoint(aws_client):
     delete_db_cluster_endpoint(db_cluster_identifier=cluster_id)
     client.delete_db_cluster_endpoint.assert_called_with(
         DBClusterEndpointIdentifier='%s.domain.endpoint' % cluster_id)
+
+
+@patch('chaosaws.rds.actions.aws_client', autospec=True)
+def test_start_db_instance(aws_client):
```

Comment:
thanks, that's useful

could we also have a test that checks for the timeout leading to the exception (which is why it's a good idea to make the timeout an argument of the action ;))